src/main/java/io/strimzi/kafka/access/KafkaAccessReconciler.java

```diff
@@ -4,6 +4,13 @@
  */
 package io.strimzi.kafka.access;
 
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
```

Comment:
Could we stick with the original order?

Reply:
done, is there any code style configuration file ?

Reply:
Thanks. Checkstyle for it is one of the things still missing ... but we can at least not change it there and back in every other PR 😉

Reply:
yeah, my ide was still configured with the setting for another project, I didn't spot it :)